riprova/retrier.py

```diff
@@ -157,19 +159,25 @@ def _handle_error(self, err):
         # Update latest cached error
         self.error = err
 
+        # Did user interrupt function?
+        if err is KeyboardInterrupt or err is ImportError:
+            raise
```

Comment:
Thanks for this, but I'm going to support this in a more versatile way where the developer can add/mutate the error exceptions whitelist.